docs/content/how-to/azure/create-self-managed-azure-cluster.md

```
@@ -113,92 +113,126 @@ GetSubnetID=$(az network vnet subnet show \
+    --assign-service-principal-roles \
+    --workload-identities-file ./workload-identities.json \
+    --diagnostics-storage-account-type Managed
+```
+
+### Configuring Azure Marketplace Images
+
+HyperShift supports multiple approaches for configuring Azure Marketplace images for your cluster nodes. The recommended approach varies based on your OpenShift version and requirements.
+
+#### For OpenShift 4.20 and Later (Recommended)
+
+**Pattern 1: Use Release Payload Defaults (Simplest)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)